chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -113,6 +113,12 @@
 
 nova = search(:node, "run_list_map:nova-controller").first
 
+unless nova.nil?
+  keystone_settings = KeystoneHelper.keystone_settings(nova, "nova")
```

Comment:
So I could do explicit search for a keystone node and check `keystone[:keystone][:ssl][:insecure]`, is that right?